src/main/java/io/r2dbc/postgresql/PostgresqlConnectionConfiguration.java

```diff
@@ -150,6 +166,10 @@ boolean isForceBinary() {
 
         private String schema;
 
+        private boolean ssl = false;
```

Comment:
We could probably add more methods to configure SSL certificates, but I think for now we might just start with default SSL properties and add all the sugar later.

Reply:
No quite sure how to use this ConnectionFactoryOptions.SSL property at all, tbh.

Reply:
`ConnectionFactoryOptions.SSL` is an indicator to enable SSL usage which means that we want to use SSL with full validation (CA, hostname, certificate validity) and fail if it's not supported/available.